docs/administration/collector-metrics-and-alerts.adoc

```diff
@@ -0,0 +1,33 @@
+= Collector Metrics and Alerts
```

Comment:
We may need to add a note here this only applies to "infra" namespaces and we have no details for collectors deployed to non-infra NS.  Unless you were able to learn if it is possible to collect metrics from non-infra namespaces by some enablement

Reply:
added in docs how to enable